src/app/src/test/kotlin/org/imperial/mrc/hint/unit/controllers/MetadataControllerTests.kt

```diff
@@ -39,4 +47,63 @@ class MetadataControllerTests
         val result = sut.version()
         assertThat(result).isSameAs(mockResponse)
     }
-}
\ No newline at end of file
+
+    @Test
+    fun `gets generic chart metadata`()
+    {
+        val tmpDir = "tmp"
```

Comment:
That's true. I'm not really sure it's worth the added complexity for a temporary implementation, particularly since we'd need to test the interface implementation separately anyway.